packages/starter/react-ui-starter/src/App.tsx

```diff
@@ -7,6 +7,7 @@ import {
     SlopeWalletAdapter,
     SolflareWalletAdapter,
     TorusWalletAdapter,
+    StreambirdWalletAdapter
```

Comment:
Please limit changes to adding the package only. We don't add all new wallets to the example or starter projects.